server/column.go

```diff
@@ -15,22 +15,30 @@ package server
 
 // ColumnInfo contains information of a column
 type ColumnInfo struct {
-	Schema             string
-	Table              string
-	OrgTable           string
-	Name               string
-	OrgName            string
-	ColumnLength       uint32
-	Charset            uint16
-	Flag               uint16
-	Decimal            uint8
-	Type               uint8
-	DefaultValueLength uint64
-	DefaultValue       []byte
+	Schema       string
+	Table        string
+	OrgTable     string
+	Name         string
+	OrgName      string
+	ColumnLength uint32
+	Charset      uint16
+	Flag         uint16
+	Decimal      uint8
+	Type         uint8
 }
 
+// DumpOpt is used to control dump execution.
+type DumpOpt int
+
+const (
+	// Normal option indicts that dump with normal steps.
+	Normal DumpOpt = 1 << iota
+	// WithDefaultValue option indicts that dump with an addition defaultValues step.
+	WithDefaultValue
+)
+
 // Dump dumps ColumnInfo to bytes.
-func (column *ColumnInfo) Dump(buffer []byte) []byte {
+func (column *ColumnInfo) Dump(buffer []byte, flags DumpOpt) []byte {
```

Comment:
Can we use a bool argument to control whether output default value?

Reply:
done~